cmd/combine/combine.go

```diff
@@ -79,16 +79,21 @@ func Combine(ctx context.Context, inputDir, outputDir string, force bool, opts .
 	var combinedKeys []tbls.PrivateKey
 
 	for idx, pkSet := range privkeys {
+		if len(pkSet) != len(lock.Operators) {
+			return errors.New(
+				"amount of private keys read doesn't match the current validator index",
+				z.Int("validator_number", idx),
+				z.Int("required_amount", len(lock.Operators)),
+				z.Int("got", len(pkSet)),
+			)
+		}
+
 		log.Info(ctx, "Recombining key share", z.Int("validator_number", idx))
```

Comment:
suggest aligning the language throughout this command: key vs key share vs private key vs private key share

Reply:
suggest: `private key share`